assets/js/modules/adsense/datastore/settings.test.js

```diff
@@ -335,5 +354,79 @@ describe( 'modules/adsense settings', () => {
 				).toEqual( value );
 			} );
 		} );
+
+		describe( 'getOriginalUseSnippet', () => {
+			it( 'uses a resolver to make a network request via getSettings', async () => {
+				const response = { useSnippet: false };
+				fetchMock.getOnce(
+					/^\/google-site-kit\/v1\/modules\/adsense\/data\/settings/,
+					{ body: response, status: 200 }
```

Comment:
We can omit `status: 200` since this is the default value:
```suggestion
					{ body: response }
```

Reply:
I think it makes more sense to have it explicitly in tests.

Reply:
Ok, let it be there then.